src/components/NavigationFloating.js

```diff
@@ -0,0 +1,145 @@
+      boxShadow: theme.custom.shadows.gray,
+      borderRadius: '16px',
+      margin: '30px 32px',
+      padding: '9px',
+      minHeight: 'auto',
+      color: theme.custom.colors.purple,
+      textAlign: 'center',
+      fontSize: '16px',
+      fontWeight: '500',
+
+      '& a': {
+        color: theme.custom.colors.purple,
+        fontSize: '16px',
+        fontWeight: '500',
+        textDecoration: 'none',
+      },
+    },
+  },
+
+  fabContainer: {
+    bottom: '15px',
+    right: '15px',
+    position: 'fixed',
+    color: theme.custom.colors.white,
+    background: theme.custom.gradients.orange,
+  },
+
+  dotsText: {
+    fontSize: '44px',
+    lineHeight: '65px',
+    position: 'relative',
+    top: '-11px',
+  },
+
+  menuNavigationIconClose: {
+    color: theme.custom.colors.white,
+    width: '19px',
+    position: 'absolute',
+    bottom: '-12px',
+    right: '20px',
+  },
+}));
+
+export default function NavigationFloating() {
+  const classes = useStyles();
+
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div>
+      <Fab
+        aria-controls={open ? 'navigation-floating-menu' : undefined}
+        aria-expanded={open ? 'true' : undefined}
+        aria-haspopup="true"
+        className={classes.fabContainer}
+        id="navigation-floating-menu-btn"
+        onClick={handleClick}
+      >
+        <span className={classes.dotsText}>...</span>
+      </Fab>
+      <Menu
+        anchorEl={anchorEl}
+        anchorOrigin={{
+          vertical: 'top',
+          horizontal: 'left',
+        }}
+        aria-labelledby="navigation-floating-menu-btn"
+        className={classes.menuNavigation}
+        id="navigation-floating-menu"
+        open={open}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'left',
+        }}
+        onClose={handleClose}
+      >
+        <MenuItem>
+          <Link to={ORGANIZATION_MEMBERS_ADD_PATH}>
+            {translate('NavigationFloating.linkAddMembers')}
+          </Link>
+        </MenuItem>
+        <MenuItem onClick={handleClose}>
+          {translate('NavigationFloating.linkEditProfile')}
+        </MenuItem>
+        <MenuItem>
+          <Link to={MY_PROFILE_PATH}>
+            {translate('NavigationFloating.linkMyWallets')}
+          </Link>
+        </MenuItem>
+        <MenuItem>
+          <ExternalLink href={MARKETPLACE_MARKET_URL}>
+            {translate('NavigationFloating.linkMarketplace')}
+          </ExternalLink>
+        </MenuItem>
+        <MenuItem onClick={handleClose}>
+          {translate('NavigationFloating.linkSupport')}
+        </MenuItem>
+        <IconClose
+          className={classes.menuNavigationIconClose}
+          onClick={handleClose}
+        />
+      </Menu>
+    </div>
+  );
+}
```

Comment:
I think we need to check with Luciaabout desktop designs in general, like the menu being like fullscreen is probably not ideal. Let's bring it up to talk about providing desktop designs in general

Reply:
I think already discussed on meeting with Lucia